examples/seldon_custom_pytorch_deployment/pytorch_seldon_deploy.py

```diff
@@ -0,0 +1,284 @@
+from torch.utils.data import DataLoader
+
+from zenml.integrations.constants import PYTORCH, SELDON
+from zenml.integrations.seldon.custom_deployer.ZenMLCustomModel import (
+    ZenMLCustomModel,
+)
+from zenml.pipelines import pipeline
+from zenml.steps import step
+from zenml.steps.base_step_config import BaseStepConfig
+
+logger = logging.getLogger(__name__)
+
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+requirements_file = os.path.join(os.path.dirname(__file__), "requirements.txt")
+
+
+class Net(nn.Module):
+    """Network architecture"""
+
+    def __init__(self):
+        super(Net, self).__init__()
+        self.conv1 = nn.Conv2d(1, 10, kernel_size=5)
+        self.conv2 = nn.Conv2d(10, 20, kernel_size=5)
+        self.conv2_drop = nn.Dropout2d()
+        self.fc1 = nn.Linear(320, 50)
+        self.fc2 = nn.Linear(50, 10)
+
+    def forward(self, x):
+        x = F.relu(F.max_pool2d(self.conv1(x), 2))
+        x = F.relu(F.max_pool2d(self.conv2_drop(self.conv2(x)), 2))
+        x = x.view(-1, 320)
+        x = F.relu(self.fc1(x))
+        x = F.dropout(x, training=self.training)
+        x = self.fc2(x)
+        return F.log_softmax(x, dim=1)
+
+
+def data_loader(
+    train_set: bool = True, batch_size: int = 4, shuffle: bool = True
+) -> DataLoader:
+    """Returns a torch Dataloader from two np arrays."""
+    data_loader = torch.utils.data.DataLoader(
+        torchvision.datasets.MNIST(
+            "/Users/safoine-zenml/work-dir/mnist/data",
+            train=train_set,
+            download=True,
+            transform=torchvision.transforms.Compose(
+                [
+                    torchvision.transforms.ToTensor(),
+                    torchvision.transforms.Normalize((0.1307,), (0.3081,)),
+                ]
+            ),
+        ),
+        batch_size=batch_size,
+        shuffle=shuffle,
+    )
+    return data_loader
+
+
+class TrainerConfig(BaseStepConfig):
+    """Trainer params"""
+
+    batch_size: int = 4
+    epochs: int = 2
+    lr: float = 0.01
+    momentum: float = 0.5
+    shuffle: bool = True
+
+
+@step()
+def torch_trainer(
+    config: TrainerConfig,
+) -> nn.Module:
+    """Train a neural net from scratch to recognize MNIST digits return our
+    model or the learner"""
+    train_loader = data_loader(
+        train_set=True, batch_size=config.batch_size, shuffle=config.shuffle
+    )
+
+    model = Net().to(DEVICE)
+    optimizer = optim.SGD(
+        model.parameters(), lr=config.lr, momentum=config.momentum
+    )
+
+    for epoch in range(1, config.epochs + 1):
+        model.train()
+        for batch_idx, (data, target) in enumerate(train_loader):
+            data, target = data.to(DEVICE), target.to(DEVICE)
+            optimizer.zero_grad()
+            output = model(data)
+            loss = F.nll_loss(output, target)
+            loss.backward()
+            optimizer.step()
+
+        print("Train Epoch: {} \tLoss: {:.6f}".format(epoch, loss.item()))
```

Comment:
```suggestion
        print("Train epoch: {} \tLoss: {:.6f}".format(epoch, loss.item()))
```

Also, (apart from when using a logger), we generally use f-strings at ZenML. You might want to replace any other places you use this style with f-strings.